Sources/GeneralIOs/Commands/Generate.swift

```diff
@@ -64,16 +73,21 @@ public final class Generate: ParsableCommand {
     }
 
     public func run() throws {
-        let renderer = Renderer(name: name,
-                                template: template,
-                                path: path,
-                                variables: [],
-                                output: output,
-                                dependencies: Services)
         if let xcodeSpec = generalSpec?.xcode {
-            guard let projectName = xcodeSpec.name ?? askProject() else {
+            guard let projectName = xcodeSpec.name ?? askProject()else {
                 throw Error.projectName
             }
+
+            var marked = generalSpec?.services.serviceMarks
+            marked?[Key.company] = xcodeSpec.company ?? askCompanyName
+            let isNewFile = !FileManager.default.fileExists(atPath: generalSpec?.services.servicesPath ?? "") ? "\(true)" : ""
```

Comment:
Let's add 
```
guard let generalSpec = generalSpec else {
    throw ...
}
```
to avoid unnecessary `??`

Also, it'd be great to cleanup isNewFile logic - `!isSomething ? true : false` is just not readable, why not do `isSomething ? false : true` instead?
Finally, using an empty string for a false feels wrong, will it handle everything correctly if we'd just path "false" there?